tests/integration_tests/build/test_coverage.py

```diff
@@ -18,6 +18,14 @@
 import host_tools.cargo_build as host  # pylint: disable=import-error
 
 COVERAGE_TARGET_PCT = 84.71
+# This slight difference comes from
+# the brand string, On Intel, it contains
+# the frequency while on AMD it does not.
+# (the cpuid crate). In the future other
+# differences may appear.
+if "AMD" in platform.processor():
```

Comment:
Why do we need different coverage on AMD?

Reply:
Coverage is different on AMD because of this function https://github.com/firecracker-microvm/firecracker/blob/master/src/cpuid/src/brand_string.rs#L84. There is exactly a difference of 6 lines (the ones that look for the frequency string in the brand string; AMD's brand string does not have that).

Reply:
Seems ok with our current method of updating the coverage. Thanks!